### PR TITLE
Add Gsheets setup guide

### PR DESCRIPTION
**Pull request by sherifnada**

### CODE REVIEW

**Review by michel-tricot — CHANGES_REQUESTED** [review]
I am super concerned about the complexity of enabling GG spreadsheet.

GG spreadsheet will most likely be synced by the least technical people and yet it seems to be the most complex one to configure.

Is this the only way? How are other services doing it?

**Review by michel-tricot — APPROVED** [review]
Looks good. but if we know a way to make the configuration actually user-friendly we should have a follow up on this work.

**Comment by sherifnada** [comment]
Created an issue capturing the work needed to improve the auth flow https://github.com/airbytehq/airbyte/issues/766

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
   * [Full Refresh](high-level-overview/full-refresh.md)
 * [Integrations](integrations/README.md)
   * [Sources](integrations/sources/README.md)
+    * [Google Sheets](integrations/sources/google-sheets.md)
```
[the remark this answers is not in the document]
Comment (sherifnada) [reply docs/SUMMARY.md]: done

**Pull request merged**